src/dashboard/src/settings/base.py

```diff
@@ -508,5 +516,17 @@
     AUTHENTICATION_BACKENDS += ["components.accounts.backends.CustomLDAPBackend"]
     from .components.ldap_auth import *  # noqa
 
+
+OIDC_AUTHENTICATION = config.get("oidc_authentication")
+if OIDC_AUTHENTICATION:
+    ALLOW_USER_EDITS = False
+
+    AUTHENTICATION_BACKENDS += ["components.accounts.backends.CustomOIDCBackend"]
+    LOGIN_EXEMPT_URLS.append(r"^oidc")
```

Comment:
You're adding `r"^oidc"` to LOGIN_EXEMPT_URLS both here and on line 411 above. Surely we don't need both?

Reply:
Oops. Yes, it should be in one or the other, not both.